sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -51,6 +52,8 @@
 # status enums for the status tracking table
 statuses = Enum("INPROGRESS", "SUCCEEDED", "FAILED", name="status")
 
+dependency_directions = Enum("UPSTREAM", "DOWNSTREAM", name="dependency_direction")
```

Comment:
It looked like in your indexer PR that you started using SqlEnum. Should you use that here instead of Enum?